### ISSUE TITLE
Does not run on JDK 13

### ISSUE THREAD

**Issue by nhartner** [graalvm/graal-js-jdk11-maven-demo#28]
Running on JDK 13 causes the following error when running `mvn package`

```=== Graal.js via org.graalvm.polyglot.Context ===
=== Graal.js via javax.script.ScriptEngine ===
ScriptEngineManager providers.next(): javax.script.ScriptEngineFactory: Provider com.oracle.truffle.js.scriptengine.GraalJSEngineFactory could not be instantiated
*** Graal.js not found ***
[ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 4.714 s <<< FAILURE! - in com.mycompany.app.AppTest
[ERROR] testGraalPolyglotSpeed(com.mycompany.app.AppTest)  Time elapsed: 0.005 s  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class org.graalvm.polyglot.Engine$ImplHolder
	at com.mycompany.app.AppTest.testGraalPolyglotSpeed(AppTest.java:59)
```

I'm running on OSX Mojave with the latest JDK from Oracle which outputs the following version info: 
```
java -version
java version "13" 2019-09-17
Java(TM) SE Runtime Environment (build 13+33)
Java HotSpot(TM) 64-Bit Server VM (build 13+33, mixed mode, sharing)
```

**Comment by nhartner** [comment]
Most likely caused by this issue https://github.com/oracle/graal/issues/1751

**Comment by nhartner** [comment]
This appears to be fixed as of graalvm library version 19.3.0
